k8s-deployer/package.json

```diff
@@ -14,9 +14,11 @@
   "devDependencies": {
     "@types/chai": "^4.3.11",
     "@types/chai-as-promised": "^7.1.8",
+    "@types/express": "^4.17.21",
     "@types/mocha": "^10.0.6",
     "@types/node": "^20.8.9",
     "@types/sinon": "^17.0.2",
+    "@types/swagger-ui-express": "^4.1.6",
```

Comment:
Is this needed for deployer?

Reply:
thanks, left over from trying something else. I have removed it now